tutorials/users/ide/version-control.md

```diff
@@ -1 +1,607 @@
+```
+(create_local_repo)=
+### Creating a Local Repository
+
+Navigate to the directory that contains your analysis or project and for which you want to implement version control. For example, if 
+your `writable-workspace` directory contains three project folders named `project_A`, `project_B`, `project_C` etc. and you want to 
+initialize version control for `project_A` then navigate to that directory. 
+
+```{caution}
+While you can configure a single local Git repository for all your data on Marble by initializing that repository in the `writable-workspace`
+directory, we strongly recommend that you initialize repositories on a per-project basis. This will make it easier for you
+to track changes that affect that project only, and it will make sharing easier by allowing you to only share the analysis 
+that you want, without exposing the work you are doing for any other project.
+```
+[](gui_git_setup)  
+[](terminal_git_setup)
+
+(gui_git_setup)=
+### Setup Git Repository Using the Graphical User Interface
+
+This section shows how to setup a Git repository using the built-in GUI, to see how to setup using a terminal see [here](terminal_git_setup). On the left sidebar click the Git icon. If there is no Git repository already set up you will see the following:
+
+![No Repository Set Menu](images/git-no-repo-menu.png)
+
+Click `Initialize a Repository` to create a Git repository in the current folder. Once this is done, the left sidebar will show the Git interface.  This is where you will see changes to files in the Git folder.
+
+![Repository Created UI](images/git-repo-created.png)
+
+(terminal_git_setup)=
+### Setup Git Repository Using the JupyterLab Terminal
+
+This section shows how to setup a Git repository using the built-in terminal. If you prefer to use a graphical interface, see [this section](gui_git_setup) instead. Click the Terminal button from the a new "Launcher" page, as shown below, (or go to **File** &rarr; **New** &rarr; **Terminal**) to start a Terminal session.
+
+![Terminal Icon](images/terminal-icon.png)
+
+![Terminal Session Screen](images/terminal-session-screen.png)
+
+Check to see that you are in the default folder (this will be the root folder, which is indicated by `/`) by executing the `ls` command.
+
+```
+ls
+```
+
+If you are in the default folder you will see the following subfolders available to you:
+
+![Terminal Default Subfolders](images/terminal-default-folders.png)
+
+Navigate into the `notebook_dir` directory.
+
+```
+cd /notebook_dir
+```
+You should see the user folders available to you.  These are the same folders seen when clicking the `File Browser` icon.
+
+![Terminal Notebook Dir](images/terminal-notebook-dir.png)
+![File Browser Default Folders](images/file-browser-default-folders.png)
+
+Navigate into the `writable-workspace` folder, then navigate to the project folder of choice (in this example `project_A`), or create a new folder
```

Comment:
You're right. I started rearranging some instructions using generic labels "project_A" etc. and I have yet to resolve these issues where some of the stuff I rearranged conflicts with stuff that Alex had before.

I was organizing content around generic project labels because I am thinking that in the intro page to JupyterLab (and before version control), we'll include some comments on "best practices" for users. There, we can talk about not putting content in "mypublic" directly and about keeping projects separate. I will use these labels "project_A" etc. in that discussion.

Sound good to you?

Reply:
sure sounds good